App.js

```diff
@@ -0,0 +1,77 @@
+import { View, ScrollView } from 'react-native'
+import React, { useState } from 'react'
+import { NavigationContainer } from '@react-navigation/native'
+import { createNativeStackNavigator } from '@react-navigation/native-stack'
+import ActionBarImage from './components/ActionBarImage'
+import Article from './components/Article'
+import ListFormats from './components/ListFormats'
+import ListCreatives from './components/ListCreatives'
+import ListIntegrations from './components/ListIntegrations'
+import { LinearGradient } from 'expo-linear-gradient'
+import ActionBarImageWhite from './components/ActionBarImageWhite'
+import styles from './styles/styles.js'
+
+const Stack = createNativeStackNavigator()
+
+function HomeScreen({ navigation }) {
+  //state for variables custom Pid, openning of the modal and selection of creative style
+  const [pid, onChangePid] = useState('84242')
+  const [modalVisible, setModalVisible] = useState(false)
+  const [selectedCreative, setSelectedCreative] = useState('Landscape')
+
+  return (
+    <ScrollView contentContainerStyle={{ flexGrow: 1 }}>
+      <View>
+        {/* Component List Of Formats */}
+        <ListFormats />
+
+        {/* List Of choice for Creatives with selector using the State */}
+        <ListCreatives
+          pid={pid}
+          onChangePid={onChangePid}
+          modalVisible={modalVisible}
+          setModalVisible={setModalVisible}
+          selectedCreative={selectedCreative}
+          setSelectedCreative={setSelectedCreative}
+        />
+
+        {/* List Of Intégrations redirecting to the demo pages */}
+        <ListIntegrations navigation={navigation} />
+      </View>
+    </ScrollView>
+  )
+}
```

Comment:
i don't know the best practices, but it is not a best practice to have only one file per component ? even if this component is static like `HomeScreen`